### ISSUE TITLE
Block of Cheese not combining

### ISSUE THREAD

**Issue by mataraki** [ValveSoftware/Dota2-Gameplay#12761]
### Item name

Block of Cheese

### Description

So, basically Cheese and 3 Greater Healing Lotuses didn't combined into a Block of Cheese in my game recently.

All of them were in backpack, we thought that might be the issue, so we dragged them up - nothing changed.

If I were to predict the bug roots: we had a guy abandoned the game at minute 5.

### Example Match ID and Timestamp

7374769520 - 51:52
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/71872467/0f7d40b9-0cae-403a-9a71-d63b09aa76fa)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/71872467/79ad5ece-59f4-48fa-9f8d-e86c6c01045a)

**Comment by commentator24** [comment]
you sure you had 3? i can only see 1

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/7f26cbec-cbc0-4219-8959-8edf130d4b1f)

**Comment by mataraki** [comment]
> you sure you had 3? i can only see 1
> 
> ![image](https://user-images.githubusercontent.com/109324155/274908564-7f26cbec-cbc0-4219-8959-8edf130d4b1f.png)

My bro, you can't see number of charges with this thing.

My bad though, could've post a screenshot. Done now.

**Comment by Philaeux** [comment]
-> Item ownership may prevent item fusing for cheese+lotuses into block of cheese
Picking up lotuses or cheese set the item as "owned" by someone. However, ownership may prevent the fusing of these items into block of cheese.

Demo: https://youtu.be/AatyGGF-JOo

**Comment by mataraki** [comment]
> -> Item ownership may prevent item fusing for cheese+lotuses into block of cheese Picking up lotuses or cheese set the item as "owned" by someone. However, ownership may prevent the fusing of these items into block of cheese.
> 
> Demo: https://youtu.be/AatyGGF-JOo

Yeah, I figured that out recently too while testing, but thanks a lot for clarification!

I don't think it's necessary at all though, especially when you have to pick 27 lotuses for this - how do you even track the "ownership"?

That's why I'm not closing the bug.

**Comment by Philaeux** [comment]
It's definitively a bug in the sense that whoever is owning this, it's in your inventory, so you should be able to fuse them.
Fusing restrictions makes sense if you pick a ogre axe for a friend in base to bring it to him, you don't want it to fuse with your items. However, for cheeses and lotuses, ownership should not stop items from fusing.
